modules/ROOT/pages/hybrid-schedule-mgmt.adoc

```diff
@@ -1,37 +1,96 @@
+* Enable or disable the Scheduler
+* Change the frequency
+* Specify when to start running the Scheduler
+* Specify the timezone (for a cron Scheduler)
 
-* Enable or disable polling elements.
-* Change the polling frequency.
 
-You can perform these tasks at runtime, without having to make changes to or redeploy the application.
+Runtime Manager lists the name of each Scheduler component as `polling://{flow_name}`.
 
 == Use Cases
 
 Schedule management is useful in the following contexts:
 
-* You want to change your schedules in your application at runtime without redeploying.
+* You want to change the schedules in your application at runtime without redeploying.
 * You want to disable a scheduled job while one of the applications you are connecting to is undergoing maintenance.
 * You want runtime visibility into the schedules in your application.
 
 
+== Considerations and Limitations
+
+When using the *Schedules* tab, consider the following:
+
+* If a scheduled job is not triggered because the application is not running,
+Runtime Manager triggers the job as soon as the application restarts.
+* The scheduler reads the job configuration every time it runs.
++
+To apply your changes immediately, run your jobs manually after updating the schedule.
+
+
+
+== View and Manage Application Schedules in Runtime Manager
+
+To use the scheduling feature in Runtime Manager, deploy an application that includes one or more Scheduler elements:
+
+.The arrow shows the *Scheduler* component in the Anypoint Studio canvas.
+image::hybrid-scheduler-studio.png[Scheduler component in the Anypoint Studio canvas]
+
+After deploying the app, view and manage the Scheduler elements in Runtime Manager:
+
+. Click *Applications* in the left pane, click the app name, and then click *Schedules*:
++
+.The screenshot shows (*1*) the *Schedules* option and (*2*) the checkbox on the *Schedules* page.
+image::hybrid-scheduler-rtm.png[Schedules option and checkbox on the Schedules page]
++
+The *Name* column shows the name of the flow that contains the Scheduler element.
++
+Runtime Manager lists the Scheduler elements in the order that they appear in your application.
+
+
+. Select the checkbox to manage a single Scheduler:
++
+--
+.The screenshot shows (*1*) the checkbox, (*2*) the *Disable* button, and (*3*) the schedule in the details pane.
+image::hybrid-scheduler-rtm-detail.png[Checkbox, Disable button, and schedule in the details pane]
+
+** Disable or enable a Scheduler.
+** Update a cron Scheduler:
++
+.The arrow shows the schedule configuration in the details pane.
+image::hybrid-scheduler-cron.png[Schedule configuration in the details pane]
+
+** Update a fixed-frequency Scheduler:
++
+.The arrow shows the schedule configuration in the details pane.
+image::hybrid-scheduler-fixed-frequency.png[Schedule configuration in the details pane]
+
+--
+. Click *Update* to save changed settings.
+. Select multiple checkboxes to enable and disable multiple schedules.
```

Comment:
I feel like this belongs up with the step 
. Select the checkbox to manage a single Scheduler:
Maybe 
. Either select one checkbox to manage a single Scheduler, or select multiple checkboxes to enable and disable (but not update) multiple schedules:

Reply:
Or we could make each goal a separate step?
. Enable one or more schedules by
. Update a cron Scheduler by
. Update a fixed-frequency Scheduler by
. Disable one or more schedules by